### ISSUE TITLE
Can't delete integration from my HA

### ISSUE THREAD

**Issue by Mercury-pl** [michaelarnauts/home-assistant-comfoconnect#79]
Hi, my Comfoconnect has a new IP so I wanted to delete old Hub and and a new one (new IP)
It seems that I can't delete it, so I thought about deleting whole integration and installing it back.
In HA when I click on the integration's options and delete option, I see the message that it will be deleted. I agree and nothing is happening. So I tried to delete it from HACS and it won't let me until I erase integration...
I tried to restart everything and still the same issue.
So I tried to add a new hub, and I get this error message:

Config flow could not be loaded: {"message":"Invalid handler specified"}

**Comment by jpdacostaza** [comment]
Having the same issue

**Comment by astyrrian1** [comment]
Same

**Comment by dbernt55** [comment]
I had the same problem. I solved it by hashing (#) all entries in the configuration yaml and deleting the comfoairconnect folder in the customcomponent folder. Restart HA, integration is gone.
Then I installed the integeation from here: https://github.com/michaelarnauts/home-assistant-comfoconnect.
That worked and I got the majority of the entities back. There are a few missing, though.
Hope that helps!
Cheers

**Comment by Witek1974** [comment]
> I had the same problem. I solved it by hashing (#) all comfoconnect entries in the configuration.yaml and deleting the comfoconnect folder in the customcomponent folder. Restart HA, integration is gone. Then I installed the integration from here: https://github.com/michaelarnauts/home-assistant-comfoconnect. That worked and I got the majority of the entities back.

Unfortunately, it doesn't work for me. 
I still get a message: 
Error
Failed to load configuration interface: 500 Internal Server Error Server got itself in trouble

**Comment by michaelarnauts** [comment]
I think your home assistant log should have more information.

**Comment by Witek1974** [comment]
> I think your home assistant log should have more information.

Indeed, there is more. The version error is strange because I am installing version v0.2.3 from HACS

Rejestrator: homeassistant.util.package
Źródło: util/package.py:150
Pierwsze zdarzenie: 21 października 2024 15:59:10 (3 zdarzenia)
Ostatnio zalogowany: 21 października 2024 15:59:12

Unable to install package aiocomfoconnect==0.1.13: error: error sending request for url (https://wheels.home-assistant.io/musllinux-index/aiocomfoconnect/) Caused by: client error (SendRequest) Caused by: connection error Caused by: received fatal alert: BadRecordMac

**Comment by michaelarnauts** [comment]
The version is correct. The latest version of the library aiocomfoconnect is 0.1.13.

What's strange is that it's trying to load that URL. I guess this has something to do with prebuild wheels for python. Something that I think has to be done for this to work on alpine Linux. But I'm not sure why I don't have this issue.

Are you on an ARM architecture, or have you installed HASS in another way (not using Home Assistant OS)?

**Comment by michaelarnauts** [comment]
I'm not an expert on this subject, but I think I publish wheels for my library: https://pypi.org/project/aiocomfoconnect/#files

Home Assistant has their own list of wheels (also for custom components): https://github.com/home-assistant/wheels-custom-integrations

But I don't understand why he our installation isn't using the published wheel, but insists on checking the home assistant list.

**Comment by Witek1974** [comment]
> Are you on an ARM architecture, or have you installed HASS in another way (not using Home Assistant OS)?

My system is Home Assistant Supervised running on Debian12, running on an AMD processor. Core 2024.10.3, Supervisor 2024.10.2.
And I don't know anything about wheels ;-)

**Comment by michaelarnauts** [comment]
The "Not loaded" state is the clue: when the setup of the entry fails, the entry gets stuck in a state where removal does not go through cleanly.

Several causes of failed setups are fixed in the pre-release: an unreachable bridge is handled and triggers a rediscovery instead of an unhandled error (#161), the fan platform no longer fails to import (#157), and adding or re-authenticating a bridge works again (#160). Since your bridge had changed IP, the rediscovery is likely the relevant one, it now updates the address in the config entry by itself.

Could you try [v0.5.0-rc1](https://github.com/michaelarnauts/home-assistant-comfoconnect/releases/tag/v0.5.0-rc1)? In HACS, turn on "Show beta versions" in the download dialog to get it. If it still happens, a debug log would help, with `custom_components.comfoconnect: debug` and `aiocomfoconnect: debug` under `logger:`.
